spec/integration/dry/files_spec.rb

```diff
@@ -11,6 +10,12 @@
     FileUtils.remove_entry_secure(root)
   end
 
+  describe "$INPUT_RECORD_SEPARATOR" do
+    it "is not nil" do
+      expect($INPUT_RECORD_SEPARATOR).not_to be_nil
```

Comment:
```suggestion
      expected = if windows?
        "\r\n"
      else
        "\n"
      end

      expect($INPUT_RECORD_SEPARATOR).to eq(expected)
```

Then we can try to run CI on windows as well, if GHA allows that.

Reply:
Based on my research "$INPUT_RECORD_SEPARATOR does not give us \r\n on Windows" (from above). I don't have a Windows computer to confirm this but that is [what I found](https://stackoverflow.com/questions/12819803/does-ruby-have-a-constant-for-cross-platform-eol-somewhere#comment60681175_12820252).

Besides, we don't need to test Ruby. This is a regression test to ensure we don't remove `require "english"` from this library (which is what this PR sought to fix in the first place). We don't care about the value, just that it's non-nil. And Windows CI sounds like something we should avoid if we can :)